server/src/main/java/org/opensearch/cluster/node/DiscoveryNode.java

```diff
@@ -94,7 +94,7 @@ public static boolean hasRole(final Settings settings, final DiscoveryNodeRole r
     }
 
     public static boolean isMasterNode(Settings settings) {
```

Comment:
isMasterNode -> isClusterManager?

Reply:
Deprecating isMaster in favour of isClusterManager sounds like a plan

Reply:
I will mark it in an issue and deprecate it another PR. Thank you!